style.css

```diff
@@ -84,11 +83,11 @@ footer {
   text-align: center;
   flex-direction: column;
   height: fit-content;
-  width: 80%;
   margin: 2rem 10%;
 }
 
-h1 {
+h1,
+#topic h2 {
   /* margin-top: 2rem; */
```

Comment:
- [ ] As mentioned in the `HTML and CSS` best practices, please do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history. 👋 😄